lib/peatio/upstream/binance.rb

```diff
@@ -18,6 +18,7 @@
 # All API provided by module is async.
 class Peatio::Upstream::Binance
   include Peatio::Bus
+  include Singleton
```

Comment:
I'm not sure that making implicit Singleton out of this class is correct approach. It may hinder testing it in future.